js/app/modules/selection/supplementaryArticles.js

```diff
@@ -12,22 +12,20 @@ const SupplementaryArticles = new Module.Class({
     Name: 'Selection.SupplementaryArticles',
     Extends: Xapian.Xapian,
 
-    // Selection.Xapian implementation
-    on_history_changed: function (history) {
-        if (!this.global)
-            return;
```

Comment:
`this.global` was already always true in the case of Selection.SupplementaryArticles.